core/vcpu.c

```diff
@@ -2015,18 +2015,11 @@ static void vmwrite_cr(struct vcpu_t *vcpu)
     }
 
     if ((state->_cr4 & CR4_PAE) && (state->_cr0 & CR0_PG) &&
-        (state->_cr0 & CR0_PE)) {
+        (state->_cr0 & CR0_PE))
```

Comment:
According to the coding style, conditional or loop statements with complex conditions or statements may be more readable with curly braces. Some projects require that an if must always have an accompanying brace.

Reply:
OK, I'll keep the curly braces.